README.md

```diff
@@ -80,7 +80,7 @@ Iceberg's Spark integration is compatible with the following Spark versions:
 | --------------- | ------------- |
 | 0.2.0+ *        | 2.3.0         |
 | 0.3.0+ *        | 2.3.2         |
-| master branch   | 2.4.0         |
+| master branch   | 2.4.4         |
```

Comment:
I think we should not drop support of 2.4.0.

Reply:
Maybe change it to `2.4.x`?